lib/app_bar_avatar.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class SetAvatar extends ConsumerWidget {
```

Comment:
Let's be consistent with naming: if filte is called app_bar_avatar widget class should be the same.

Reply:
Done